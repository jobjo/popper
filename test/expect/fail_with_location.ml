open Popper

type int_list = int list [@@deriving show, ord, popper]

let test_rev =
  test @@ fun () ->
  equal ~loc:__LOC__ int_list_comparator (List.rev [ 1; 2; 3 ]) [ 2; 3; 1 ]

let () = Clean.run test_rev
