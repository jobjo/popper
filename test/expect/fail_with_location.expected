

FAIL: 0/1 tests passed and 1 failed in XXs.

  ✖  Anonymous  Failed

┌────────────────────────────────────────────────────────────────────────────────────────┐
│ Failed                                                                                 │
└────────────────────────────────────────────────────────────────────────────────────────┘

  Reason:

    [3; 2; 1] <> [2; 3; 1]

  Location:

    File "test/expect/fail_with_location.ml", line 7, characters 13-20

